packages/core/test/CSL/testData.ts

```diff
@@ -30,6 +30,7 @@ export const poolParameters: Cardano.PoolParameters = {
   vrf
 };
 
+// REVIEW: Also add these additional assets here?
```

Comment:
Good idea :+1:

Reply:
Done in [9cb225b](https://github.com/input-output-hk/cardano-js-sdk/pull/274/commits/9cb225be0fea0ec2da65f393f7120be986f0929d)